src/thread.cpp

```diff
@@ -155,11 +170,21 @@ void Node::workerThread(int number) {
         break;
       }
     }
+    logDeque = thd->get_recent_queries();
```

Comment:
Will this work ? in the case of mode pquery. you are not saving query.

Reply:
hmm, removed, this will not work and as usual give 1-2 queries even if i ran --lo-all-queries